### PR TITLE
Adding routes

### PR DESCRIPTION
**Pull request by BrennaManning**
Adding routes in index.js file
Each route currently sending example html file.

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -6,4 +6,46 @@ router.get('/', function(req, res, next) {
...
+	res.sendfile('./public/index.html');
+
+});
+
+router.get('/student/add', function(req, res){
+	//Add new student
+	res.sendfile('./public/index.html');
+});
+
+router.get('/student/edit', function(req, res){
```
Comment (aconsilvio) [inline routes/index.js]: can we change some of the gets to posts?

File: routes/index.js [hunk 2]
```diff
@@ -6,4 +6,46 @@ router.get('/', function(req, res, next) {
...
+	//GET program
+	res.sendfile('./public/index.html');
+});
+
+router.get('/student', function(req, res){
+	//GET student
+	res.sendfile('./public/index.html');
+
+});
+
```
Comment (nmohamed) [inline routes/index.js:42]: this is an empty space ;-)

**Comment by shrutiyer** [comment]
Looks good

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<html>
```
Comment (davidabrahams) [inline public/index.html]: this file can be deleted (I think)

File: routes/index.js [hunk 4]
```diff
@@ -1,9 +1,58 @@
...
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
   res.render('index', { title: 'Express' });
 });
 
+router.post('/login', function(req, res){
+	//POST login
+	res.sendfile('./views/index.html');
```
Comment (davidabrahams) [inline routes/index.js]: According to [this](http://stackoverflow.com/questions/25463423/res-sendfile-absolute-path) stackoverflow: `res.sendfile` is deprecated. I think the currently supported way to do it is:

```
res.sendFile('index.html', { root: path.join(__dirname, '../views') });
```

I think this should work?

**Comment by davidabrahams** [comment]
Just had two comments, everything else looks good, and I ran it locally and it works!

**Comment by nmohamed** [comment]
:+1:

File: models/studentDataModel.js [hunk 5]
```diff
@@ -1,14 +1,13 @@
...
 	type: String,
 	date: Date,
 	data: Array
-});  
-​
+});
+
 var studentSchema = mongoose.Schema({
 	id: Number,
 	name: String,
```
Comment (nmohamed) [inline models/studentDataModel.js:13]: for line 15, - 17:
make it attendance: Array

**Pull request merged**